bpf/lib/common.h

```diff
@@ -616,8 +618,8 @@ struct lb6_service {
 	};
 	__u16 count;
 	__u16 rev_nat_index;
-	__u8 flags;
-	__u8 pad[3];
+	__u16 flags;
```

Comment:
Doesn't this break agent up/downgrades given old/new bits could be at different locations now?

Reply:
I'll look into the upgrade implications. Thanks.

Reply:
Just to add more context, two examples what this might break:
- An externalIP SVC is allowed to be accessed during upgrade (until bpf_sock.o has been reloaded) due to https://github.com/cilium/cilium/blob/master/bpf/bpf_sock.c#L200 failing to detect that the SVC is of the "externalIP" type.
- A ClusterIP service is allowed to be accessed from outside, as `nodeport_lb4(...)` is considering the SVC as routable.